09/test.md

```diff
@@ -0,0 +1,244 @@
+
+- Enlista al menos 3 meta tags con su uso y su sintaxis
+
+> Respuesta:`<h1>`para poner algun emcabezado `<ol>`para hacer un lista en orden y `<img>` para añadir una imagen al documento
+
+- ¿Cuántas etiquetas `<h>` existen?
+
+> Respuesta: Desde el `<h1>`hasta el `<h6>`
+
+- ¿Cuál es la diferencia entre `<h>` y `<p>`?
+
+> Respuesta: El `<h>` se usa par los emcabezados y el `<p>`se usa para agregar parrafos
+
+- ¿Cuál es la diferencia entre `<p>` y `<span>`?
+
+> Respuesta: El `<p>`es para poner un parrafo y el `<span>`para dar estilo
+
+- Enlista al menos 2 etiquetas usadas para modificar el estilo del texto
+
+> Respuesta: `<strong>`-`<em>` y `<span>`
+
+- Enlista todas las etiquetas de contenedores vistas en el curso
+
+> Respuesta: `<header>`-`<nav>`-`<section>`-`<aside>`-`<article>`-`<footer>`
+
+- ¿Qué es HTML semántico?
+
+> Respuesta: Es una estructura de codigo para organizar una pagina web de manera que sea mas legible
+
+- ¿Cuál es la diferencia entre `<ul>` y `<ol>`
+
+> Respuesta: `<ul>`para una lista sin orden y `<ol>`para una lista ordenada
+
+- ¿Qué es un input?
+
+> Respuesta: Los inputs se usan para crear controles ,botones o  para obtener informacion de un usuario en un formulario
+
+- Escribe un ejemplo de los siguientes inputs
+
+> Respuesta:
+> Checkbox -> `<input type="checkbox">`
+> Date -> `<input type="Date">`
+> File -> `<input type="file>`
+> Password -> `<input type="password"`
+> Radio -> `<input type="radio">`
+> Range -> `<input type="range">`
+> Text -> `<input type="text"`
+> Textarea -> `<textarea>Escribe</textarea>`
+> Select -> `<select><option>Opcion1</option><option>Opcion2</option></select>`
+
+- ¿Cómo podemos poner un valor por defecto en cualquier input?
+
+> Respuesta: el valor checked o value
+
+- ¿Cuál es el uso de la etiqueta `<label>`?
+
+> Respuesta: darle un nombre a nuestro input
+
+- ¿Cómo hacemos un link entre un input y su label?
+
+> Respuesta: Con el ID del label y del input
+
+- ¿Cómo le damos cierta ayuda al usuario en los inputs?
+
+> Respuesta: con el placeholder
+
+- ¿Cómo podemos bloquear un input?
+
+> Respuesta: Con el valor disabled
+
+- ¿Dentro de cuál etiqueta deberían de estar todos los inputs?
+
+> Respuesta: Con la etiqueta legend
+
+- ¿Cómo podemos crear secciones dentro de los formularios?
+
+> Respuesta: con la etiqueta fieldset
+
+- Escribe un ejemplo de cada etiqueta
+
+> Respuesta:
+> Imagen -> `<img src="carpetaImagenes/imagen1">`
+> Audio -> `<audio src="carpetaMusica/music4">`
+> Video -> `<video src="carpetaVideo/VIDEO9">`
+> Recursos Externos (maps, web sites) -> `<iframe src="https://www.apple.com">`
+
+- ¿Qué atributos usamos para darle tamaño a una imagen?
+
+> Respuesta: el width y height
+
+- ¿Cómo ponemos etiquetas de emergencia para archivos de imágenes, audio o video?
+
+> Respuesta: con el atributo alt
```

Comment:
This is not the correct answer. the idea is to let the browser decide which format is valid in each caso
No es la respuesta correcta. La idea es permitir al explorador decidir cuál formato es válido para cada caso